### PR TITLE
Upgrade angular

### PR DESCRIPTION
**Pull request by cgrigis**

### CODE REVIEW

File: webapp/package.json [hunk 1]
```diff
@@ -13,19 +13,19 @@
...
     "@angular/material": "^7.3.7",
-    "@angular/platform-browser": "^7.2.15",
-    "@angular/platform-browser-dynamic": "^7.2.15",
-    "@angular/router": "^7.2.15",
+    "@angular/platform-browser": "^8.2.10",
+    "@angular/platform-browser-dynamic": "^8.2.10",
+    "@angular/router": "^8.2.10",
     "@c4dt/cothority": "^3.1.23",
-    "@c4dt/dynacred": "dev",
+    "@c4dt/dynacred": "0.1.1-p20191010083636.0",
```
Comment (tharvik) [inline webapp/package.json]: You might want to keep using the "dev" tag (yes, it gets replaced at each update, thanks npm)

File: webapp/tsconfig.json [hunk 2]
```diff
@@ -2,17 +2,18 @@
   "compileOnSave": false,
   "compilerOptions": {
     "baseUrl": "./",
+    "downlevelIteration": true,
```
Comment (tharvik) [inline webapp/tsconfig.json]: From [the release notes from typescript](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-3.html#new---downleveliteration), if you use `target >= es2015`, you don't need it.

File: webapp/package.json (hunk 1 above)
Comment (cgrigis) [reply webapp/package.json]: Right, it's probably the "~automatic Angular upgrade procedure" that changed it...

Comment (cgrigis) [reply webapp/package.json]: This being said, shouldn't we hardcode the version and upgrade it manually, in order to have reproducible builds?

File: webapp/tsconfig.json (hunk 2 above)
Comment (cgrigis) [reply webapp/tsconfig.json]: An automatic change done by the Angular upgrade script, along with the change of target. :-/

**Pull request merged**